src/kzg10/mod.rs

```diff
@@ -41,6 +45,8 @@ where
         produce_g2_powers: bool,
         rng: &mut R,
     ) -> Result<UniversalParams<E>, Error> {
+        let produce_h_powers = true;
```

Comment:
Let's take this as a parameter to the method (like `produce_g2_powers`)

Reply:
Yeah, I did this as a placeholder so I didn't change the existing places that called `setup`